src/components/Footer/Footer.module.css

```diff
@@ -34,6 +33,10 @@
     font-size: 2rem;
```

Comment:
Footer size shouldn't be handled here, are we missing the same from Typography component

Reply:
I'm currently reviewing the main changes, including the Typography component for compliance.

Reply:
@poulami-saha, I have updated the HeroFooter component. Typography included